### PR TITLE
Question-85: Positional Encoding for Transformer Inputs

### PR DESCRIPTION
**Pull request by dsaha21**
Hi @moe18 

Adding the Position Encodings with input embeddings required for Transformer model. 

This section will go to NLP ( medium). 

Question mentioned in Issue #165

### CODE REVIEW

**Comment by moe18** [comment]
Hey @dsaha21, would you be able to convert your learn section into markdown similar to this https://github.com/Open-Deep-ML/DML-OpenProblem/blob/main/example_problem/learn.md, thank you and sorry for the confusion as we are trying to go from html to markdown, let me know if you have any issues or questions

**Comment by dsaha21** [comment]
Hi @moe18, 
I made the following changes as requested. Please review once and let me know if any other changes are required.

Thanks

File: Problems/Positional_encoding/solution.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    pos_encoding = pos_encoding[np.newaxis, :]
+    pos_encoding.shape
+    pos_encoding = np.float32(pos_encoding)
+    
+    return pos_encoding
+
+def test_pos_encoding() -> None:
+    # Test case 1:
+    ans1 = (1, 2, 8)
+    assert pos_encoding(2,8).shape == ans1, "Test case 1 failed"
```
Comment (moe18) [inline Problems/Positional_encoding/solution.py:30]: the tests should test what result people got from pos_encoding(2,8) and not the shape

**Review by moe18 — COMMENTED** [review]
Looks like a good problem, but why are you testing the shape of the result and not the result itself?

File: Problems/Positional_encoding/solution.py (hunk 1 above)
Comment (dsaha21) [reply Problems/Positional_encoding/solution.py:30]: Ok @moe18, will update the code section asap. Thanks

**Comment by dsaha21** [comment]
Hi @moe18 , I have made the possible changes. 
Please let me know if this is fine. 
Thanks :)

**Review by Open-Deep-ML — APPROVED** [review]
looks good

**Comment by Open-Deep-ML** [comment]
could you make it question 85

**Comment by moe18** [comment]
the folder should be called 
Problems/85_Positional_encoding/

**Comment by dsaha21** [comment]
Please check once @moe18, I have updated it. If everything's fine we can close this. 

Thanks

**Pull request merged**